db/index.js

```diff
@@ -0,0 +1,16 @@
+const mysql = require('mysql');
+
+const database = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password17',
```

Comment:
Highly reccomend you put this in a config.js and export the variable, and put config.js on your .gitignore